elasticai/creator/nn/quantized_grads/fixed_point/_two_complement_fixed_point_config.py

```diff
@@ -1,13 +1,16 @@
 from dataclasses import dataclass
+from typing import Optional
 
 import torch
 
+from elasticai.creator.nn.quantized_grads.quantization_config import QuantizationConfig
+
 
 @dataclass(frozen=True)
-class FixedPointConfigV2:
+class FixedPointConfigV2(QuantizationConfig):
     total_bits: int
     frac_bits: int
-    stochastic_rounding: bool
+    stochastic_rounding: Optional[bool] = False
```

Comment:
With optional you're saying that this may be None. Also note: this syntax for optionals is deprecated (and now i understand why :D), we should use `bool | None` instead.